packages/xdl/src/Web.js

```diff
@@ -2,14 +2,44 @@ import opn from 'opn';
 
 import Logger from './Logger';
 import * as UrlUtils from './UrlUtils';
+import chalk from 'chalk';
 
-export async function openProjectAsync(projectRoot) {
+export async function openProjectAsync(projectRoot, options) {
   try {
-    let url = await UrlUtils.constructWebAppUrlAsync(projectRoot);
+    let url = await UrlUtils.constructWebAppUrlAsync(projectRoot, options);
     opn(url, { wait: false });
     return { success: true, url };
   } catch (e) {
     Logger.global.error(`Couldn't start project on web: ${e.message}`);
     return { success: false, error: e };
   }
 }
+
+export function getWebSetupLogs() {
+  const appJsonRules = chalk.white(
+    `
+  ${chalk.whiteBright.bold(`app.json`)}
+  {
+    "packages": [
```

Comment:
I think this should be `"platforms": ["android", "ios", "web"]` (this is what we check in https://github.com/expo/expo-cli/blob/9137987338f5a4dbc372bbfba6f2653df2d7a3d5/packages/expo-cli/src/commands/start/TerminalUI.js#L146).

Reply:
The fields is `platforms`, not `packages`.
```suggestion
    "platforms": [
```